pkg/cmd/pulumi/preview.go

```diff
@@ -200,6 +207,26 @@ func newPreviewCmd() *cobra.Command {
 			case expectNop && changes != nil && changes.HasChanges():
 				return result.FromError(errors.New("error: no changes were expected but changes were proposed"))
 			default:
+				if planFilePath != "" {
+					encrypter, err := sm.Encrypter()
+					if err != nil {
+						return result.FromError(err)
+					}
+					if err = writePlan(planFilePath, plan, encrypter, showSecrets); err != nil {
+						return result.FromError(err)
+					}
+
+					// Write out message on how to use the plan
+					var buf bytes.Buffer
+					writer := bufio.NewWriter(&buf)
```

Comment:
Nit: no need to wrap this in a `bufio.Writer`--you can just use `&buf` directly.